<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connectway</class>
 <widget class="QMainWindow" name="Connectway">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBoxserial">
          <property name="styleSheet">
           <string notr="true">font: 600 11pt &quot;Segoe UI Semibold&quot;;</string>
          </property>
          <property name="text">
           <string>Serial</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="scanPushButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>10</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	font: 10pt &quot;楷体&quot;;
    border: 2px solid#cccccc;
    border-radius: 10px;
    padding: 5px 15px;
   background-color: rgb(255, 255, 255);
}QPushButton:hover {
            background-color: rgb(184, 184, 184); 
        }</string>
            </property>
            <property name="text">
             <string>扫描串口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="openserialpushButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	font: 10pt &quot;楷体&quot;;
    border: 2px solid#cccccc;
    border-radius: 10px;
    padding: 5px 15px;
   background-color: rgb(255, 255, 255);
}QPushButton:hover {
   background-color: rgb(184, 184, 184);
        }</string>
            </property>
            <property name="text">
             <string>打开串口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="outserialpushButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	font: 10pt &quot;楷体&quot;;
    border: 2px solid#cccccc;
    border-radius: 10px;
    padding: 5px 15px;
   background-color: rgb(255, 255, 255);
}QPushButton:hover {
   background-color: rgb(184, 184, 184);
        }</string>
            </property>
            <property name="text">
             <string>关闭串口</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="serialPortLabel">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>串口名：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="serialPortComboBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="baudRateLabel">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>波特率：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="baudRateComboBox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>2400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>14400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>115200</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="stopLabel">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>停止位：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="stopComboBox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1.5</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="dataLabel">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>数据位：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="dataComboBox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="checkLabel">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>校验位：</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="checkComboBox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Odd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Even</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkBoxinter">
            <property name="styleSheet">
             <string notr="true">font: 600 11pt &quot;Segoe UI Semibold&quot;;</string>
            </property>
            <property name="text">
             <string>Inter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="interconpushButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	font: 10pt &quot;楷体&quot;;
    border: 2px solid#cccccc;
    border-radius: 10px;
    padding: 5px 15px;
   background-color: rgb(255, 255, 255);
}QPushButton:hover {
   background-color: rgb(184, 184, 184);
        }</string>
            </property>
            <property name="text">
             <string>连接</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="outinterpushButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	font: 10pt &quot;楷体&quot;;
    border: 2px solid#cccccc;
    border-radius: 10px;
    padding: 5px 15px;
   background-color: rgb(255, 255, 255);
}QPushButton:hover {
   background-color: rgb(184, 184, 184);
        }</string>
            </property>
            <property name="text">
             <string>断开</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
          <item>
           <widget class="QLabel" name="IPlabel">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">	font: 13pt &quot;楷体&quot;;</string>
            </property>
            <property name="text">
             <string>服务端IP: </string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="IPlineEdit">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>220</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
    border-radius: 10px;
    padding: 10px;
    border: 2px solid #b7e3ff;
    background-color: #ffffff;
    /* 可选：设置字体和颜色 */
   	font: 10pt &quot;幼圆&quot;;
    color: #333333;
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="daunkoulabel">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">	font: 13pt &quot;楷体&quot;;</string>
            </property>
            <property name="text">
             <string>端口号: </string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="duankoulineEdit">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>220</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
    border-radius: 10px;
    padding: 10px;
    border: 2px solid #b7e3ff;
    background-color: #ffffff;
    /* 可选：设置字体和颜色 */
   	font: 10pt &quot;幼圆&quot;;
    color: #333333;
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>447</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
